build.py

```diff
@@ -46,49 +48,12 @@ def _set(instance):
 
     return _set
 
-def set_font_metaData(font, sort):
```

Comment:
I believe @aaronbell chose to do this so that we didn't have to do it for a whole heap of different UFO fontinfos, but I'm not certain. It does seem like we're trading DRY for correctness by having it in the build script :smile: which I don't entirely mind

Reply:
Yeah, while I don't mind updating the metadata in the fontinfos, I would prefer a single source of truth in the build process, and that would be in build.py. That way should will always know that those values are set as desired, and nothing will impact them (such as any roundtrips through font editing software).

Reply:
My thinking was that you're using Glyphs to do the design work anyway, and so you have a single source of truth. GlyphsLib takes care to sync the data across all UFOs. You can also use Glyphs' custom parameters on instances to change stuff.

Reply:
I would agree with you if we were using GlyphsLib as part of build.py. But since we're not, it seems strange to me to be reliant on converting from Glyphs every time I want to modify the version string. One might say that it is easy enough to go into the files and modify the UFO directly in such a situation, but introduce a manual step when I can automate it via the build file :).

Reply:
Hm!!! Are you not spending the majority of time in Glyphs? I mean sure, I can reinstate the function with version setting?

Reply:
I mean, *I* am. And I do use GlyphsLib to roundtrip. 

But that doesn't mean that I will always be the one building / modifying the font. In a number of cases, Dustin has updated the version string, but he's on a Windows device and it doesn't make sense to force him to do that manually 😄 .

Anyway, I think I'd prefer to keep the Version info, metrics, and GASP table info here. Version info for the reason above. Metrics because I like being able to see them set clearly here (so no matter what happens to the sources in the future, they are locked to what they should be in the official build script), and GASP because I prefer to have VTT-level control over what the GASP table looks like rather than being reliant on Glyphs' implementation.

Reply:
Note: the gasp UFO stuff is the same thing as the actual `gasp` table, so you aren't relying on anything Glyphs here.